crates/client/src/config.rs

```diff
@@ -94,8 +209,80 @@ impl Config {
         }
         throw!(Error::BadWorkspace)
     }
+    pub fn get_env_variable(self, cli_input: String) -> String {
+        // Tested on a few examples, HFUZZ_RUN_ARGS give precedence to the later arguments.
+        // so if HFUZZ_RUN_ARGS="-t 10 -t 15" -> timeout 15s is applied.
+        // That means we do not need to parse the arguments from the CLI;
+        // thus, we can simply append them at the end, and the CLI will have precedence.
+
+        let mut toml_vars: String = String::new();
+        for x in self.fuzz.fuzz_args {
```

Comment:
I find this for loop not very readable and also not very idiomatic. Also, the code could eventually panic if  an argument without short and long options is created (it is not currently the case, but it is possible). The whole function body could be replaced by for example this:

```rust
let mut args: Vec<String> = self
            .fuzz
            .fuzz_args
            .iter()
            .map(|a| {
                let val = a.val.to_owned().unwrap_or("".to_string());
                if let Some(o) = &a.short_opt {
                    format!("{} {}", o, val)
                } else if let Some(o) = &a.long_opt {
                    format!("{} {}", o, val)
                } else {
                    "".to_string()
                }
            })
            .collect();
        args.push(cli_input);
        args.join(" ")
```